easybuild/framework/easyconfig/default.py

```diff
@@ -88,9 +88,11 @@
                                       "to be linked in any installed binary/library", BUILD],
     'bitbucket_account': ['%(namelower)s', "Bitbucket account name to be used to resolve template values in source"
                                            " URLs", BUILD],
-    'buildopts': ['', 'Extra options passed to make step (default already has -j X)', BUILD],
+    'buildopts': ['', 'Extra options appended to build command', BUILD],
+    'build_cmd': [None, "Main shell command used in the build step", BUILD],
```

Comment:
This should also mention something like "`(only used by default build step implemented by EasyBlock class)`", or something like that?
As soon as a generic/custom easyblock is used, `build_cmd` will be totally ignored.
Same goes for `configure`, `test`, `install`